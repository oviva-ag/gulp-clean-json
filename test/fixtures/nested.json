{
    "hello": "",
    "some": {
        "deep": {
            "another": 5,
            "like": {
                "really": {
                    "deep": {
                        "thing": ""
                    }
                },
                "a": {
                    "lot": {
                        "thing": null,
                        "maybe": {}
                    }
                }
            }
        }
    }
}