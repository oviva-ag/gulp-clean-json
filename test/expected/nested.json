{
	"some": {
		"deep": {
			"another": 5
		}
	}
}